src/fprime_gds/common/pipeline/files.py

```diff
@@ -43,10 +43,11 @@ def setup_file_handling(
         )
         file_decoder.register(self.__downlinker)
         distributor.register("FW_PACKET_HAND", self.__uplinker)
-        if not os.access(down_store, os.W_OK):
+        try:
+            Path(down_store).mkdir(parents=True, exist_ok=True)
+        except PermissionError:
```

Comment:
Changing the way we check for permissions here because `os.access(down_store, os.W_OK)` returns False if down_store doesn't exist, even if user could create down_store and have permissions on it. Which is really what we're trying to test here. This is already handled in StandardPipeline.setup(), but trying to make this test more robust / agnostic to StandardPipeline.